app/src/main/java/de/othr/plantico/ui/ownedPlantList/OwnedPlantListAdapter.kt

```diff
@@ -0,0 +1,163 @@
+
+                    WateringLevel.LOW -> {
+                        itemBinding.wateringLight.setImageDrawable(waterdropplettFull)
+                        itemBinding.wateringMedium.setImageDrawable(waterdropplettEmpty)
+                        itemBinding.wateringHeavy.setImageDrawable(waterdropplettEmpty)
+                    }
+                    WateringLevel.MEDIUM -> {
+                        itemBinding.wateringLight.setImageDrawable(waterdropplettFull)
+                        itemBinding.wateringMedium.setImageDrawable(waterdropplettFull)
+                        itemBinding.wateringHeavy.setImageDrawable(waterdropplettEmpty)
+                    }
+                    WateringLevel.HIGH -> {
+                        itemBinding.wateringLight.setImageDrawable(waterdropplettFull)
+                        itemBinding.wateringMedium.setImageDrawable(waterdropplettFull)
+                        itemBinding.wateringHeavy.setImageDrawable(waterdropplettFull)
+                    }
+                }
+
+                val currentDate = Date().nowUTC()
+
+                if (plant.lastWatered != null) {
+
+
+                    var wateringDate = plant.lastWatered
+                    wateringDate = if (plant.customWateringCycle != null) {
+                        wateringDate.addDays(plant.customWateringCycle)
+                    } else {
+                        wateringDate.addDays(realPlant.wateringCycleDays)
+                    }
+                    val diff: Long = wateringDate.time - currentDate.time
+                    val days = (((diff / 1000) / 60) / 60) / 24
+                    itemBinding.plantWateringTimeText.text = "$days days"
+
+                }
+
+                if(plant.birthday != null){
+                    val currentDate = Date().nowUTC()
+
+                    val diff: Long = currentDate.time - plant.birthday.time
+                    val seconds = diff / 1000
+                    val minutes = seconds / 60
+                    val hours = minutes / 60
+                    val days = hours / 24
+                    val years = days / 365
+
+                    val ageText = if(years>0){
+                        "$years years old"
+                    }else{
+                        "$days days old"
+                    }
+                    itemBinding.plantAgeText.text = ageText
+                }
+                else{
+                    itemBinding.plantAgeText.text = "-"
+                }
+            }
+
+        }
+
+        override fun onClick(p0: View?) {
+            if (p0 != null) {
+                if (p0.id == R.id.watering_card) {
+                    val context = p0.context
+                    val intent = Intent(context, PlantActivity::class.java).putExtra(
+                        PlantActivity.SELECTED_PLANT,
+                        currentList[layoutPosition].id
+                    )
+                    context.startActivity(intent)
+                }
+            }
+        }
```

Comment:
Please check this. As far as I can tell this doesn't go to the Plant page. For example watch at the wateringAdapter. There it is working.